features/commands.js

```diff
@@ -305,6 +305,23 @@ Here's an article explaining the difference between the two: https://goshakkk.na
 
         fetchMsg.delete();
       }
+    },
+    {
+      words: [`!appIdeas`],
```

Comment:
```suggestion
      words: [`!appideas`],
```I looked over our current code again, and realized that the capital `I` here could break our case sensitivity checks, because the trigger isn't lowercased while comparing against the lowercased message. Making this change would allow both `!appideas` and `!appIdeas`